.github/workflows/conda-build-test.yml

```
@@ -1,10 +1,13 @@
 name: conda builds
 
 on:
-  push:
-    branches: [master]
   pull_request:
-  workflow_dispatch:
+    paths:
+      - '.github/workflows/conda-build-test.yml'
+      - 'src/admin-guide/environments/*.ya?ml'
+  schedule:
+    - cron: '17 22 * * 6'
+
```

Keep the workflow dispatch event - it's always useful
```suggestion
  workflow_dispatch
```